bundle/Resources/config/ezplatform.yml

```diff
@@ -0,0 +1,6 @@
+system:
+    default:
+        field_templates:
+            - { template: "@NetgenEzPlatformSiteApi/content_fields.html.twig", priority: 0 }
+        fielddefinition_settings_templates:
+            - { template: "@NetgenEzPlatformSiteApi/field_definition_settings_templates.html.twig", priority: 0 }
```

Comment:
Wrong template file name, there's an extra `_templates`.

Reply:
Fixed in https://github.com/netgen/ezplatform-site-api/pull/94/commits/ce2aaafb65d50c921519c8ffb3ffda1500537d62